generators/generator-bot-adaptive/baseGenerator.js

```diff
@@ -35,7 +35,14 @@ module.exports = class extends Generator {
       alias: 'i',
     });
 
-    const { botName, platform, integration } = this.options;
+    this.option('sdkVersion', {
+      desc: 'The Bot Framework SDK version to use',
+      type: String,
+      default: platforms.dotnet_sdk,
```

Comment:
I'd actually suggest setting a default value in the language-specific code blocks below. Then you don't have to introduce logic to ensure that a platform selection of `js` actually changes the default value to the `js_sdk` value.

For .NET, you would do:
```javascript
sdkVersion: this.options.sdkVersion || platforms.dotnet_sdk,
```

For JS, you would do:
```javascript
const { botName, integration, sdkVersion = platforms.js_sdk } = this.options;
```

Reply:
Ah.... Good catch.  Specifying a platform of "js", but not the SDK version would not work as expected in my initial version.